bluesky/tests/test_examples.py

```diff
@@ -451,13 +451,13 @@ def test_duplicate_keys(RE, hw):
     # two detectors, same data keys
 
     def gen():
-        yield(Msg('open_run'))
-        yield(Msg('create', name='primary'))
-        yield(Msg('trigger', hw.det))
-        yield(Msg('trigger', hw.identical_det))
-        yield(Msg('read', hw.det))
-        yield(Msg('read', hw.identical_det))
-        yield(Msg('save'))
+        yield (Msg('open_run'))
```

Comment:
Does it need the parentheses around `Msg` in this case?

Reply:
No, but they do no harm.

Reply:
No _technical_ harm.  They do cause minor obfuscation, such as this question.  Better to remove if not strictly needed.